### DISCUSSION TITLE
Config and carousel

### DISCUSSION THREAD

**Discussion by AnaGiachini** [mdbootstrap/TW-Elements#2059] [Support from community]
Hello, I am requesting help for two themes, in my configuration file I had to change this plugins: [require("tw-elements-react/dist/plugin.cjs")] to plugins: [import ("tw-elements-react/dist /plugin.cjs")] because I couldn't use require. which gave me the following No declaration file found for module 'tw-elements-react/dist/plugin.cjs'. '/Users/analiagiachini/Documents/ProyectosVS/DeRaizGestoria/node_modules/tw-elements-react/dist/plugin.cjs' has an implicit type of "any".
  If package 'tw-elements-react' actually exposes this module, try adding a new declaration file (.d.ts) containing 'declare module 'tw-elements-react/dist/plugin.cjs';` and when I want to use the Carousel it gives me the following error Uncaught SyntaxError: The requested module '/node_modules/.vite/deps/tw-elements-react.js?v=82fc61a3' does not provide an export named 'TECarousel'

**Comment by juujisai** [comment]
Hi!

1. Those are not errors in terminal / app but just a messages inside your IDE right? (like the 3 dots in VSC). The plugin will work correctly. No need to do anything with that.

2. I guess you are using nextJS? If you are experiencing those issues, please use dynamic imports. Checkout out next integration tutorial: https://tw-elements.com/docs/react/integrations/next-integration/#section-tailwind  - step 7

**Comment by AnaGiachini** [comment]
Thanks! I was able to solve it
